src/main.py

```diff
@@ -50,6 +51,27 @@
 if 'debug' not in config:
 	config['debug'] = debug
 
+# Setup event commands
```

Comment:
Or we trim the values like this:
```
config_event_commands = {k: v.strip() for k, v in config['platforms']['common']['event_commands'].items()}
event_commands.update(config_event_commands)
```

but I'm not sure if that wouldn't be over-complicating things.